queries.sql

```diff
@@ -1,3 +1,43 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+-- List the name of all animals born between 2016 and 2019.
+
+SELECT name
+FROM animals
+WHERE
+    date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+
```

Comment:
- [ ] It must list the name of all animals born between 2016 and 2019. you should consider all the days of 2019 and not only the first day of January 2019. Your code might looks like this:

```SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';```
or
```SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;```

Reply:
Hi @BenMukebo 

Thank you for your valuable feedback. I shall make the required changes.

Regards